app/views/usergroups/_form.html.erb

```diff
@@ -6,6 +6,7 @@
     <% if AuthSource.non_internal.present? %>
       <li><a href="#external" data-toggle="tab"><%= _("External Groups") %></a></li>
     <% end %>
+    <%= render_tab_header_for(:usergroup_tabs, :subject => @usergroup, :form => f) %>
```

Comment:
I wonder, if it needs to be usergroup_tabs / user_tabs. In all other sections it is used like `%= render_tab_header_for(:main_tabs`  with `:main_tabs`

Reply:
I misunderstood the documentation [here](https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc#adding-a-pagelet).

Applied the change.